frameworks/kafka/src/main/dist/server.properties.mustache

```diff
@@ -19,6 +19,10 @@
 # The id of the broker. This must be set to a unique integer for each broker.
 broker.id={{POD_INSTANCE_INDEX}}
 
+{{#DETECT_ZONES}}
+broker.rack={{ZONE}}
+{{/DETECT_ZONES}}
+
```

Comment:
Is there any default value for `broker.rack`? that can be used when `detect_zones` is misconfigured or set to false or the underlying cluster does not have rack awareness support?

Reply:
https://github.com/mesosphere/dcos-commons/pull/1908/commits/07e6b7ad45dff168e87c305131a53a8e7b1a541a